docs/faq/network/validator.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Running a Validator
+description: LUKSO Validator FAQ
+sidebar_position: 2
+---
+
+:::info
+Regarding questions about validator hardware and configuration: LUKSO uses the same technology as the Ethereum network. Therefore, guides and resources on how to run a solo Ethereum validator node also apply to the LUKSO network. Here are some handy resources we recommend:
+
+- [https://www.coincashew.com/coins/overview-eth/guide-or-how-to-setup-a-validator-on-eth2-mainnet](https://www.coincashew.com/coins/overview-eth/guide-or-how-to-setup-a-validator-on-eth2-mainnet)
+- [https://ethereum.org/en/staking/solo/](https://ethereum.org/en/staking/solo/)
+- [https://www.reddit.com/r/ethstaker/](https://www.reddit.com/r/ethstaker/)
+
+:::
+
+### How many LYX are needed to run a validator?
+
+32 LYX per validator. A single node can run multiple validators.
+
+### Is there a minimum number of validators required to run a node?
+
+No, you can run as many or as few validators as you want and there is no set minimum or maximum. However, in the interest of decentralization we do not recommend running too many validators on one machine as it increases the likelihood of all of them being penalized if goes your machine goes offline.
+
+### Can I withdraw my staked LYX?
+
+LUKSO will not launch with the Shapella upgrade enabled so there will be some time when staked tokens will be unable to be withdrawn. After the team are happy with the stability of the network and the number of validators, we will support the upgrade Shapella to enable token withdrawals.
+
+### What is the easiest way of participating as a validator if I have limited technical knowledge?
+
+It should be noted that running a validator is a commitment and requires a degree of technical proficiency to maintain and set up correctly. People with limited technical knowledge may be more comfortable waiting for third party staking services to come to LUKSO before running a validator node.
+
+In the future we aim to work with third party services to make setting up solo validator nodes easier with minimal setup. However, this will not be available at genesis as the network will need to be running for some time before third parties are able to integrate.
```

Comment:
```suggestion
In the future, we aim to work with third-party services to make setting up solo validator nodes easier with minimal setup. However, this will not be available at genesis as the network must run for some time before third parties can integrate.
```